content/blog/grpc-js-1.0.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "@gprc/grpc-js 1.0 release announcement"
+date: 2020-04-20
+author:
+  name: Michael Lumish
+  link: https://github.com/murgatroid99
+---
+
+Today we are excited to announce the release of version 1.0 of [@grpc/grpc-js](https://www.npmjs.com/package/@grpc/grpc-js). This library is a reimplementation of the original [grpc](https://www.npmjs.com/package/grpc) library for Node.js, but purely in TypeScript, no longer depending on native C++ code that requires complex compilation and installation procedures.
+
+With this release, the original grpc library is deprecated. We will continue releasing new binary packages for new versions of the Node and Electron runtimes, but it will not receive any new features.
+
+We worked hard to bring @grpc/grpc-js to you, with a feature set that we believe should fit most, if not all usage cases:
```

Comment:
How about:

> ## Features
> 
> gRPC-JS supports the following features, which should cover most use cases:
>
> - Clients
> - ...

(It doesn't make sense to claim "if not all ... cases" since further below you're admitting that some of the original Node gRPC features haven't been ported over yet. :) )